draft-foudil-securitytxt.md

```diff
@@ -94,11 +94,16 @@ This document defines a text file to be placed in a known location
 that provides information about the vulnerability disclosure practices of a particular organization.
 This is intended to help security researchers when disclosing security vulnerabilities.
 
-The file is named "security.txt", and this file MUST be placed under the
-/.well-known/ path ("/.well-known/security.txt") {{!RFC8615}} of a domain name or IP address for web
-properties. For legacy compatibility, a security.txt file might be placed at the top level path (see {{weblocation}}).
+By convention, the file is named "security.txt".
 
-For web-based services, the file MUST be accessible via the Hypertext Transfer Protocol (HTTP) 1.0 {{!RFC1945}} or higher version, as a resource of Internet Media Type "text/plain" with the default charset parameter set to "utf-8" per section 4.1.3 of {{!RFC2046}}, and it MUST be served with "https" (as per section 2.7.2 of {{!RFC7230}}). For file systems a "security.txt" file SHOULD be placed in the root directory of a particular file system.
+When made available on HTTP servers, it MUST be placed under the
+/.well-known/ path (as "/.well-known/security.txt") {{!RFC8615}} of a domain name or IP address.
+For legacy compatibility, a security.txt file might be placed at the top level path (see {{weblocation}}).
+For file systems a "security.txt" file SHOULD be placed in the root directory of a particular file system.
+
+On HTTP servers, the file MUST be accessed via the Hypertext Transfer Protocol (HTTP) 1.0 {{!RFC1945}} or higher version
+and the "https" scheme (as per section 2.7.2 of {{!RFC7230}}). It MUST have a Content-Type of "text/plain"
```

Comment:
> On HTTP servers, the file MUST be accessed via the Hypertext Transfer Protocol 1.0 or a higher version and the "https" scheme.

This sentence is quite difficult to follow. Especially with all the inline references. Might need rephrasing.

Reply:
reworded